doc/error-handling/error-handling.rst

```diff
@@ -320,3 +321,24 @@ If the error was **Excessive Comm Current (code 6066)**, this means the telemetr
 If one actuator was failing, it would imply the overcurrent would be seen on only one cable.
 Therefore, if you test one cable (or zone) at one time, you should be able to identify which group of actuators might have the power issue.
 But if the error kept the same for all 3 zones, it would imply a failure in every one of 3 zones, which was highly unlikely to be the case, and you might need to check something else that might result in these errors.
+
+.. _lsst.ts.m2gui-recover_system_from_rigid_body_movement:
+
+Recover the System from the Rigid Body Movement
+-----------------------------------------------
+
+The available range of rigid body momement is restricted to the assembly and it would induce the error codes: 6055 and 6088 if the movement is big.
+Actually, in the normal telescope survey, the M2 should stay at the origin with the minimum stress.
+However, in some cases, we would need to adjust the mirror's position and this might trigger the error to transition the system to leave the closed-loop control to protect the mirror.
+
+If the error code is 6055, you can check the :ref:`lsst.ts.m2gui-user_detailed_force` to see which actuator has the force higher than the threshold of closed-loop control.
+Usually, the axial actuator is related to the z-direction and the tangent link is related to the x- and y-direction.
+Once the actuator is identified, you can move it in steps to have the force lower than the threshold of closed-loop control, and then, put the system into the closed-loop control to recover the system.
+It is noted that the single actuator movement will increase the stress of the system.
+Therefore, the actuator's force should not differ from the threshold too much before transitioning to the closed-loop control.
+Otherwise, the system might be damaged from the high stress.
```

Comment:
I would add also to check the 4 extra conditions on the tangent links.

Reply:
Added in the commit: [2fbe0cc](https://github.com/lsst-ts/ts_m2gui/pull/55/commits/2fbe0ccb8c2407391d367a675d2b3a2c410b8fb1). Thanks!